composer.json

```diff
@@ -22,8 +22,7 @@
     "require-dev" : {
         "phpunit/phpunit": "^8.0",
         "guzzlehttp/psr7": "^1.5.2",
-        "php-http/guzzle7-adapter": "^0.1.1",
-        "http-interop/http-factory-guzzle": "^1.0"
+        "guzzlehttp/guzzle": "^7.2"
```

Comment:
No, you need http-interop/http-factory-guzzle, but not guzzlehttp/psr7.

Reply:
Ah :) I will fix that